src/plugins/irc-events/message.js

```diff
@@ -10,6 +10,11 @@ module.exports = function(irc, network) {
 			return;
 		}
 
+		if (data.message.substring(0,4) == '?OTR') {
+			// Hide OTR messages
+			return;
+		}
```

Comment:
"yep" was enough :D
I'm fine with anything (wouldn't want to be the one accused of [bikeshedding](https://en.wikipedia.org/wiki/Parkinson%27s_law_of_triviality), would I?), even with using the hardcoded `?OTR` here rather than a helper for this PR, so :+1: with note to move OTR intelligence over there.